process/block/preprocess/transactions.go

```diff
@@ -1334,6 +1349,60 @@ func SortTransactionsBySenderAndNonce(transactions []*txcache.WrappedTransaction
 	sort.Slice(transactions, sorter)
 }
 
+// preFilterTransactions filters the transactions prioritising the move balance operations
+func preFilterTransactions(
+	transactions []*txcache.WrappedTransaction,
+	gasBandwidth uint64,
+	gasHandler process.GasHandler,
+) []*txcache.WrappedTransaction {
+	selectedTxs := make([]*txcache.WrappedTransaction, 0, len(transactions))
+	skippedTxs := make([]*txcache.WrappedTransaction, 0)
```

Comment:
could have pre-allocated the skippedTxs also with len(transactions) as we do not know how many we skip or select (they are slices of pointers so we shouldn't worry about OOMs)